### DISCUSSION TITLE
Gomplate 3.11.6 for Alpine

### DISCUSSION THREAD

**Discussion by kenfinnigan** [hairyhenderson/gomplate#1923] [General]
Will there be an update to the [gomplate APK](https://pkgs.org/download/gomplate) for Alpine to 3.11.6 to resolve the fixed CVEs?

Is there a timeframe of when there will be a release, as 3.11.6 was released 2 weeks ago?

Thanks

**Comment by hairyhenderson** [comment]
The Alpine PR merged 5 days ago: https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/55581

It's available in `edge`: https://pkgs.alpinelinux.org/packages?name=gomplate&branch=edge&repo=&arch=&maintainer=

**Reply by kenfinnigan** [reply]
Thanks @hairyhenderson
